portfolio/src/main/webapp/style.css

```
@@ -1,9 +1,18 @@
+
 #content {
   margin-left: auto;
   margin-right: auto;
   width: 650px;
+  font-family: sans-serif
 }
 
-#greeting-container {
+#fun-fact-container {
   margin-top: 20px;
 }
+
+.centered {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
```

```suggestion
  margin: 0 auto;
```

Combines both attributes into one. The '0' means top and bottom are 0px, the 'auto' sets left and right. Same thing applies above